docs/changes/3.1.2.rst

```diff
@@ -0,0 +1,35 @@
+Changelog for Falcon 3.1.1
+==========================
+
+Summary
+-------
+
+This is a minor point release fixing a couple of high impact bugs,
+as well as publishing binary wheels for the recently released CPython 3.12.
+
+
+Changes to Supported Platforms
+------------------------------
+
+- Falcon is now supported (including binary wheels) on CPython 3.12.
+  A couple of remaining stdlib deprecations from 3.11 and 3.12 will be
+  addressed in Falcon 4.0.
+- As with the previous release, Python 3.5 & 3.6 remain deprecated and
+  will no longer be supported in Falcon 4.0.
+- EOL Python 3.7 will no longer be actively supported in 4.0, but the framework
```

Comment:
for me it's fine to also remove 3.7 support for v4, but I don't think it's high impact since there aren't much improvements.

3.8 would be higher impact since targeting 3.9+ allows the `type | type` typing syntax to work. But it's not yet that time :)